lib/Form/Type/FieldType/ImageFieldType.php

```diff
@@ -38,13 +38,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 TextType::class,
                 [
                     'label' => /** @Desc("Alternative text") */ 'content.field_type.ezimage.alternative_text',
-                    'required' => false,
+                    'required' => $options['is_alternative_text_required'],
```

Comment:
Is it guaranteed that the key is present in the array?

Reply:
As part of `configureOptions`, yes.

`setDefaults` implies that the field is required, at the same time providing a default value.